terraform/modules/services/ecs_task/variables.tf

```diff
@@ -128,6 +131,14 @@ variable ecs-task_size {
     )
     error_message = "Invalid combination of CPU and memory."
   }
+  validation {
+    condition = var.ecs-task_size.storage >= 21
+    error_message = "Storage must be at least 21."
+  }
+  validation {
+    condition = var.ecs-task_size.storage <= 200
```

Comment:
"at most" should be `<`, this fails for 200 so not inclusive?

Reply:
The logic is correct but the wording is wrong, it can be at most 200GiB: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition#ephemeral_storage

Reply:
Although if something is "at most" something I think I read that as inclusive anyway?

Reply:
Sorry yep my bad, I now read it as inclusive too 😂  as you were...